alerts.php

```diff
@@ -340,8 +340,13 @@ function ExtTransports($obj) {
             $msg        = FormatAlertTpl($obj);
             $obj['msg'] = $msg;
             echo $transport.' => ';
-            eval('$tmp = function($obj,$opts) { global $config; '.file_get_contents($config['install_dir'].'/includes/alerts/transport.'.$transport.'.php').' return false; };');
-            $tmp = $tmp($obj,$opts);
+            if ($transport == "canopsis") {
```

Comment:
No, no particular reason. I just tried to use fully qualified class names in the transport code but when I took a look in the AMQPConnection class, I saw that used namespace which is a PHP functionnality that I don't use. I keep on receiving error while trying to use this. I can try again to take a look in that way but namespace class is a thing that I don't masterize at all... :)

```
# cat /usr/share/php/PhpAmqpLib/Connection/AMQPConnection.php
<?php
namespace PhpAmqpLib\Connection;

/**
 * Class AMQPConnection
 *
 * Kept for BC
 *
 * @deprecated
 */
class AMQPConnection extends AMQPStreamConnection
{
}
```